Makefile

```diff
@@ -61,6 +61,7 @@ swagger-oss-gen: $(SWAGGER) ## Generate API Swagger specification
 	rm -f $(SPEC_TARGET)
 	SWAGGER_GENERATE_EXTENSION=false $(SWAGGER) generate spec -q -m -w pkg/server -o $(SPEC_TARGET) \
 	-x "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions" \
+	-x "github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2/options" \
```

Comment:
@grafana/backend-platform we could use you help here. Is this fix ok?

We were getting this error when generating swagger files:

`classifier: unknown swagger annotation "extendee"`

Seems related to https://github.com/go-swagger/go-swagger/issues/2917

Due to: https://github.com/grafana/authlib/blob/main/authz/proto/v1/openapi.pb.go#L10